src/btest.cr

```diff
@@ -408,21 +339,22 @@ if !Dir.exists?(config.test_path)
   Process.exit(EXIT_TEST_PATH_MISSING)
 end
 
-
 arg_threads = 1
 arg_suite = nil
 
 OptionParser.parse! do |parser|
   parser.banner = "Usage: btest [arguments]"
-  parser.on("-s SUITE", "--suite", "Run only this suite") \
-    { |s| arg_suite = s }
-  parser.on("-j THREADS", "--threads", "Use this many threads") \
-    { |t| arg_threads = t.to_i64 }
-  parser.on("-h", "--help", "Show this help") \
-    { puts parser; Process.exit(0) }
+  parser.on("-s SUITE", "--suite", "Run only this suite") { |s|
```

Comment:
use `do ... end` instead of `{ ... }` as you go multiline, it feels better on the eye (at least for me ... and the convention)